build.properties

```diff
@@ -0,0 +1,7 @@
+meiversion.latest=5.0
+meiversion.next=5.1-dev
+saxon.edition.version=SaxonHE12-5
+saxon.version=12.5
+schematron.version=8.0.0
+stylesheets.version=7.57.1
+xerces.version=26.1.0.1
```

Comment:
Please always add a new line at eof, see https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b

Reply:
Thanks for the hint. Tried to add it, hope it worked?